ncm-iptables/src/main/perl/iptables.pod

```diff
@@ -10,15 +10,11 @@ iptables: Setup the IPTABLES firewall rules.
 =head1 DESCRIPTION
 
 The I<IPTABLES> component perform the setup of the
-B</etc/sysconfig/iptables> configuration file and restarts the
+C<< /etc/sysconfig/iptables >> configuration file and restarts the
 iptables service.
 
 =head1 SYNOPSIS
 
-Note: a detailed HOWTO for this component, including examples, is
-found in /usr/share/doc/ncm-iptables-2.3.13/ncm-iptables-howto.html and in the FIO twiki pages
```

Comment:
the declaration file is now schema, but i have no clue what the functions are about.